keplergl_cli/keplergl_cli.py

```diff
@@ -40,14 +40,21 @@ def __init__(
             names=None,
             read_only=False,
             api_key=None,
-            style=None):
+            style=None,
```

Comment:
8 params is too many to allow to be positional imo. I'd rather not change the existing API but we should add `*` before `config_file` to make the new params keyword-only